source/common/common/token_bucket.h

```diff
@@ -10,28 +12,30 @@ namespace Envoy {
  * https://en.wikipedia.org/wiki/Token_bucket
  */
 class TokenBucket {
-
-  constexpr static double nano = double(std::micro::num) / double(std::micro::den);
-
 public:
   /**
    * @param max_tokens supplies the maximun number of tokens in the bucket.
-   * @param refill_rate supplies the number of tokens coming back into the bucket on each second.
+   * @param fill_rate supplies the number of tokens that will return to the bucket on each second;
+   * Default is 1 token/second.
```

Comment:
I don't see this default anywhere.

Reply:
`double fill_rate = 1`

https://github.com/gsagula/envoy/blob/b649c549159196dc1f8cf318831d7731f8278fc8/source/common/common/token_bucket.h#L23

Reply:
OK